src/js/components/lottoPurchasedList.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+import { $, $$ } from '../utils/dom.js';
+import { template } from '../utils/templates.js';
+import { lottoStore } from './store.js';
+
+class LottoPurchasedList {
+  constructor() {
+    this.$lottoPurchasedSection = $('#lotto-purchased-section');
+    this.$lottoPurcasedList = $('#lotto-purchased-list');
+    this.$toggleButton = $('.lotto-numbers-toggle-button');
+    this.$toggleButton.addEventListener('click', this.onClickToggleButton);
+  }
+
+  init(numberOfLotto) {
+    this.$lottoPurchasedSection.style.display = 'block';
+    this.createLottoList(numberOfLotto);
+    this.renderLottoList();
+  }
+
+  createLottoList(numberOfLotto) {
+    for (let i = 0; i < numberOfLotto; i++) {
+      const lottoNumbers = [];
+      for (let j = 0; j < 6; j++) {
+        const number = this.getRandomNumber(lottoNumbers);
+        lottoNumbers.push(number);
+      }
+      lottoStore.addLotto(lottoNumbers);
+    }
```

Comment:
```suggestion
[...Array(numberOfLotto)].forEach(() => {
   const lottoNumbers = [];
  [...Array(6)].forEach(() => {
     const number = this.getRandomNumber(lottoNumbers);
     lottoNumbers.push(number);
  })
  lottoStore.addLotto(lottoNumbers);
})
```

일반 for 반복문의 경우 변수 선언, 부등호 비교, 증감식 등 사람이 실수할 수 있는 요소가 많습니다. 그래서 가능하면 Array내장객체의 메서드를 이용해서 반복하는 방법이 있습니다. depth가 많이 깊어지기 때문에 `[...Array(6)]~` 이 부분을 따로 분리해보는 것은 어떨까요?

Reply:
for문, 심지어 이중 for문을 사용하면서 불안했는데...🥲
내장메서드로 수정해보겠습니다!

Reply:
cc106dd
조언해주신대로 `[...Array()].forEach()` 를 활용하여 순회를 하도록 수정했습니다 .또한 랜덤숫자 1개를 획득하는 함수와 랜덤숫자리스트를 획득하는 함수로 분리하여 depth를 낮췄습니다!

`[...Array()].forEach()` 이 문법 정말 신세계인것 같아요!! 대박입니다! 👍